Library/Formula/gdub.rb

```diff
@@ -0,0 +1,12 @@
+class Gdub < Formula
+  desc "Gdub is a Gradle wrapper wrapper"
+  homepage "http://gdub.rocks"
+  url "https://github.com/dougborg/gdub/archive/v0.1.0.tar.gz"
+  sha256 "ddf2572cc67b8df3293b1707720c6ef09d6caf73227fa869a73b16239df959c3"
+
+  depends_on "gradle"
+
+  def install
+    bin.install "bin/gw"
+  end
```

Comment:
Might need `ENV["_JAVA_OPTIONS"] = "-Duser.home=#{testpath}"` at the top of `test do` given it's throwing out to `gradle` and may be trying to write places it shouldn't be.